influxql/ast.go

```diff
@@ -372,7 +372,7 @@ func (s *CreateUserStatement) String() string {
 	_, _ = buf.WriteString("CREATE USER ")
 	_, _ = buf.WriteString(s.Name)
 	_, _ = buf.WriteString(" WITH PASSWORD ")
-	_, _ = buf.WriteString(s.Password)
```

Comment:
The create user and set password statements are not rewritten and the statement is serialized in protobuf when sent to other nodes so it should be ok.